### PR TITLE
Model Interface

### PR DESCRIPTION
**Pull request by smira-kacst**
Added an interface for the model along with updates to the dockerfile to reflect the new modifications. I found out that flask is not really needed as we can just run a python script and connect it to the cvd repository utilizing the built-in functionality of rabbitmq itself.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM python:3.6
+COPY model/matched_individuals.npy .
+COPY model/main.py .
+COPY model/src/methods src/methods
```
Comment (Almenea) [inline Dockerfile:4]: You can do 
`COPY model/src/* src/`

File: interface.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "dead": "400",
+        "recovered": "4000",
+        "susceptible": "4000"
+    }
+    channel.basic_publish(exchange='', routing_key="cces.cvd.result", body=json.dumps(sample_result))
+    print("Sent result to cces.cvd.result queue")
+
+
+channel.basic_consume(
+    queue="cces.cvd.task", on_message_callback=callback, auto_ack=True)
```
Comment (Almenea) [inline interface.py:31]: Can you try `auto_ack=False` and do it manually after that task complete successfully.

File: interface.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    print("Running model")
+    time.sleep(4)
+    sample_result = {
+        "time": str(time.localtime()),
+        "infected": "400",
+        "dead": "400",
+        "recovered": "4000",
+        "susceptible": "4000"
+    }
+    channel.basic_publish(exchange='', routing_key="cces.cvd.result", body=json.dumps(sample_result))
```
Comment (Almenea) [inline interface.py:26]: The exchange is `sims-exchange`

File: Dockerfile (hunk 1 above)
Comment (smira-kacst) [reply Dockerfile]: I originally had it this way because I did not want to copy over the cache files, fixed it now. Thanks for pointing it out 👍

**Pull request merged**